http://sourceware.org/ml/gdb-cvs/2011-05/msg00119.html

### src/gdb/ChangeLog	2011/05/16 02:22:34	1.13011
### src/gdb/ChangeLog	2011/05/16 16:33:49	1.13012
## -1,3 +1,8 @@
+2011-05-16  Doug Evans  <dje@google.com>
+
+	* python/py-auto-load.c (source_section_scripts): Mention objfile
+	name in warning.
+
 2011-05-15  Doug Evans  <dje@google.com>
 
 	* linux-thread-db.c (try_thread_db_load_from_pdir_1): New function.
Index: gdb-7.2/gdb/python/py-auto-load.c
===================================================================
--- gdb-7.2.orig/gdb/python/py-auto-load.c
+++ gdb-7.2/gdb/python/py-auto-load.c
@@ -304,9 +304,10 @@ source_section_scripts (struct objfile *
 	  /* We don't throw an error, the program is still debuggable.  */
 	  if (! pspace_info->script_not_found_warning_printed)
 	    {
-	      warning (_("Missing auto-load scripts referenced in %s.\n\
+	      warning (_("Missing auto-load scripts referenced in section %s\n\
+of file %s\n\
 Use `info auto-load-scripts [REGEXP]' to list them."),
-		       GDBPY_AUTO_SECTION_NAME);
+		       GDBPY_AUTO_SECTION_NAME, objfile->name);
 	      pspace_info->script_not_found_warning_printed = TRUE;
 	    }
 	  continue;
